src/generic.jl

```diff
@@ -1359,7 +1359,13 @@ _iszero(V) = iszero(V)
 # A Base.FastContiguousSubArray view of a StridedArray
 FastContiguousSubArrayStrided{T,N,P<:StridedArray,I<:Tuple{AbstractUnitRange, Vararg{Any}}} = Base.SubArray{T,N,P,I,true}
 # using mapreduce instead of all permits vectorization
```

Comment:
This comment should be deleted? Do we still get vectorization?

Reply:
I've updated the comment. We still get vectorization, and the performance is identical.